corehq/apps/case_importer/tasks.py

```diff
@@ -87,6 +95,20 @@ def _merge_import_results(result_list):
     return result
 
 
+def _create_deprecated_error_dict(case_type):
```

Comment:
I'd move this to maybe a classmethod on the `_ImportResults` class, since it is an import result